functions/code-suggestion/src/type.ts

```diff
@@ -2,3 +2,8 @@ export interface IUICodeSnippet {
   code: string;
   model: string;
 }
+export interface ChatSnippet {
```

Comment:
NIT: could you standardise the interface name. perhaps `IUIChatSnippet` or `IChatSnippet`